python/mathsys/dev/latex/code.py

```
@@ -0,0 +1,178 @@
+
+
+#^
+#^  LATEX
+#^
+
+#> LATEX -> GENERATOR
+class LaTeX:
+    #~ GENERATOR -> INIT
+    def __init__(self) -> None: pass
+    #~ GENERATOR -> RUN
+    def run(self, start: parser.Start) -> str: types.clear(); return self.start(start)
```

Ruff (0.14.6)

Multiple statements on one line (semicolon)

(E702)